src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    static final int ZERO_INDEX = 0;
+    static final int FIRST_INDEX = 1;
+    static final String CSV_SEPARATOR = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String readString = readFromFile(fromFileName);
+        String reportString = createReport(readString);
+        writeToFile(reportString, toFileName);
+    }
+
+    public String readFromFile(String fileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                stringBuilder.append(line).append(System.lineSeparator());
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file " + fileName, e);
+        }
+        return stringBuilder.toString();
+    }
+
+    public String createReport(String inputString) {
+        String[] records = inputString.split(System.lineSeparator());
+        int supplySum = 0;
+        int buySum = 0;
+        for (String record : records) {
+            String[] splitted = record.split(CSV_SEPARATOR);
+            if ("supply".equals(splitted[ZERO_INDEX])) {
+                supplySum += Integer.parseInt(splitted[FIRST_INDEX]);
+            } else if ("buy".equals(splitted[ZERO_INDEX])) {
+                buySum += Integer.parseInt(splitted[FIRST_INDEX]);
+            }
+        }
+        StringBuilder tempStr = new StringBuilder();
```

Comment:
**NOT FIXED**

![image](https://user-images.githubusercontent.com/20326858/211285211-4b1ba185-2ea7-49ca-9c46-13a1241b8ba1.png)